win32/devdoc/sysinfo_win32_requirements.md

```diff
@@ -19,6 +19,6 @@ MOCKABLE_FUNCTION(, uint32_t, sysinfo_get_processor_count);
 
 `sysinfo_get_processor_count` returns the processor count as returned by the operating system.
 
-**SRS_SYSINFO_WIN32_01_001: [** `sysinfo_get_processor_count` shall call `GetSystemInfo` to obtain the system information. **]**
+**SRS_SYSINFO_WIN32_43_001: [** `sysinfo_get_processor_count` shall call `GetActiveProcessorCount(ALL_PROCESSOR_GROUPS)` to obtain the number of processors. **]**
 
-**SRS_SYSINFO_WIN32_01_002: [** `sysinfo_get_processor_count` shall return the processor count as returned by `GetSystemInfo`. **]**
+**SRS_SYSINFO_WIN32_43_002: [** `sysinfo_get_processor_count` shall return the processor count as returned by `GetActiveProcessorCount`. **]**
```

Comment:
>sysinfo_get_processor_count` shall return the p [](http://example.com/codeflow?start=33&length=47)

Interestingly nothing is mentioned about failure cases.

GetActiveProcessorCount can fail, so we should at least spec what we return when it does fail:

"If the function fails, the return value is zero. To get extended error information, use GetLastError."